lib/command/index.js

```diff
@@ -372,11 +372,21 @@ class Command extends Registry {
       const arg = toQuestion(flag, current, answers)
       const res = await this.prompt(arg)
       Object.assign(answers, res)
+      // If the flag has a validation function, call it now to possibly terminate early.
+      // Pass the shape of `this.parsed` to match the parameter from non-interactive mode.
+      if (typeof current.validate === 'function') {
+        const is_valid = current.validate.call(this, {...this.parsed, ...answers})
```

Comment:
I don't think you need to call the original manually, `validate` is also called by the `prompt()` call via inquirer.

Reply:
No, it's not a need, just thought that validating after the user entered a particular value is a "feature". Otherwise, it waits for them to painfully enter all the things before erroring on one.  And the one that errors could have been the first one :)

I can change it if you'd like, but didn't seem like it hurt.

Reply:
oh, `by the prompt call via inquirer` was the problem I ran into!  It felt more like a bug because:

* It requires a return value of `true` when it felt nicer to just return `false/string` for failure.
* It calls the `validate` function early, and there's no value in it.  It's also promise-based, where I found myself in an infinite loop, even though my validator function was simple.  Not sure why.

Reply:
I added an issue for other people to track the bug (and fix): https://github.com/esatterwhite/node-seeli/issues/134